src/Mvc/test/Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -117,7 +117,7 @@ public async Task CompilationFailuresFromViewImportsAreListed()
             // Arrange
             var expectedMessage = "The type or namespace name &#x27;NamespaceDoesNotExist&#x27; could not be found ("
                 + "are you missing a using directive or an assembly reference?)";
-            var expectedCompilationContent = "public class Views_ErrorFromViewImports_Index : "
+            var expectedCompilationContent = "Views_ErrorFromViewImports_Index : "
```

Comment:
This is another bit of weirdness, when running in the publish directory, the name of the generated class has a leading _, i.e. "_Views_", so a hack is trimming the expected to eliminate the public class part...  @pranavkm any ideas why this difference might occur when running from the publish directory (as opposed to in the sources)

Reply:
It's possible that the ContentRoot is missing a trailing slash. for e.g

```
Content Root: /foo/bar
View: /foo/bar/Views/Index.cshtml

Relative path: /Views/Index.cshtml
```

which would result in the underscore.

Reply:
Ok I will try setting the content root to ~/ and see if this is fixed